bin/appscale

```diff
@@ -176,6 +176,12 @@ elif command == "register":
 elif command in ["--version", "-v"]:
   print APPSCALE_VERSION
   sys.exit(0)
+elif command == "upgrade":
+  try:
+    appscale.upgrade()
+  except Exception, e:
```

Comment:
Please replace variable _e_ with something like _exception_ throughout this file.